config/locales/en.yml

```diff
@@ -1785,6 +1785,13 @@ en:
           - To check the status of your %{current_tax_year} tax return you can log into your <a href='https://www.irs.gov/payments/your-online-account'>IRS online account</a>.
           - If you need additional support, you can reach out to a Volunteer Income Tax Assistance (VITA) site or the Taxpayer Advocate Service.
           title: Unfortunately, you can’t use GetCTC because you already filed a %{current_tax_year} tax return.
+        puerto_rico_sign_up:
+          help_text_html: |
+            <p>GetCTC is currently live for residents of the 50 states and D.C. and will be available for residents of Puerto Rico later this year.</p>
```

Comment:
Its weird that one paragraph is in a p tag and the other one isn't. The structure of this will be:

```
<p>
  <p>GetCTC is currently live ...</p>
  Get notified...
</p>
```

If we want both to be in their own p tag, we should either remove the p tag from the layout and add a p tag to the "Get notified" test in the yml OR, preferably, create a help_text_items array with text-only yml and loop through it from the template:

```
<% t("help_text_items").each do |item| %>
  <p><%= item %></p>
<% end %>
```

Reply:
oops good catch! i originally didn't notice the two paragraphs so this was a bit of an oversight